playwright-e2e/dss/enum.ts

```diff
@@ -0,0 +1,15 @@
+export enum ORGANIZATIONS {
```

Comment:
This file might want to be renamed to `enums.ts` to match the similar DSM file.

Reply:
will do!